test/test_cuda.py

```diff
@@ -4702,6 +4703,89 @@ class TestCudaOptims(TestCase):
     # These tests will be instantiate with instantiate_device_type_tests
     # to apply the new OptimizerInfo structure.
 
+    @unittest.skipIf(
+        not TEST_CUDA_GRAPH, "CUDA >= 11.0 or ROCM >= 5.3 required for graphs"
+    )
```

Comment:
```suggestion
    @onlyNativeDeviceTypes
```

why are we adding this skip?

You should be able to just copy the test case exactly...